packages/mml-viewer/src/ui/fields.ts

```diff
@@ -169,6 +169,14 @@ export const environmentMapField: FieldDefinition = {
   groupDefinition: environmentGroup,
 };
 
+export const showEnvironmentMapField: FieldDefinition = {
+  name: "showEnvironmentMapField",
+  label: "Show Environment Map",
+  type: "boolean",
+  defaultValue: false,
```

Comment:
This needs to default to `"true"` to retain backwards compatibility. The problem then becomes that you need to actually apply the default as currently defaults are assumed to be falsy / empty.

I just applied this locally and it works. Add this to the bottom of the `UIField` constructor:

```
if (fieldDefinition.type === "boolean") {
  this.setValue(fieldDefinition.defaultValue.toString());
}
```

This is so the checkbox shows as checked in the list in the UI even though you haven't specified the query param.

Reply:
derp, makes sense. ty for the feedback, addressing now.